### ISSUE TITLE
Compatibility Issue: @imgly/background-removal with Node.js v20.11.1

### ISSUE THREAD

**Issue by black-turtle** [imgly/background-removal-js#119]
Hello Guys,
I'm trying to install the `@imgly/background-removal` package (version 1.5.0) using `npm install @imgly/background-removal`. However, I'm encountering an error message indicating a potential compatibility issue with my current Node.js version.

### Error Message:

```bash
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@imgly/background-removal@1.5.0',
npm WARN EBADENGINE   required: { node: '=20.0.0' },
npm WARN EBADENGINE   current: { node: 'v20.11.1', npm: '10.2.4' }
npm WARN EBADENGINE }
```

### My Environment:
- Node.js version: v20.11.1
- npm version: 10.2.4

According to the error message, it seems the @imgly/background-removal package requires Node.js version 20.0.0 exactly. Could you please clarify if this is indeed the case or if the package might be compatible with newer Node.js versions like mine (v20.11.1)? If there's any documentation or compatibility information available, it would be very helpful.

**Comment by pinksLi** [comment]
Me too
Request help

Hello Guys,
I'm trying to install the @imgly/background-removal package (version 1.5.0) using npm install @imgly/background-removal. However, I'm encountering an error message indicating a potential compatibility issue with my current Node.js version.



npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@imgly/background-removal@1.5.0',
npm WARN EBADENGINE   required: { node: '=20.0.0' },
npm WARN EBADENGINE   current: { node: 'v16..18.1', npm: '8.19.2' }
npm WARN EBADENGINE }



<img width="450" alt="去除背景依赖的node版本2" src="https://github.com/imgly/background-removal-js/assets/87258222/133aa452-f694-4202-b024-f60145dd3333">

**Comment by pinksLi** [comment]
import { removeBackground } from "@imgly/background-removal"

**Comment by pinksLi** [comment]
import { removeBackground } from "@imgly/background-removal"

**Comment by pinksLi** [comment]
Solution
Use the following methods

import { removeBackground } from "@imgly/background-removal"
let firstImg = ref()
let lastImg = ref()
function removeImgBack() {
    removeBackground(firstImg.value, {}).then((blob) => {
      lastImg.value = URL.createObjectURL(blob)
    })
  }

**Comment by pinksLi** [comment]
Solution
Use the following methods

import { removeBackground } from "@imgly/background-removal"
let firstImg = ref()
let lastImg = ref()
function removeImgBack() {
  removeBackground(firstImg.value, {}).then((blob) => {
  lastImg.value = URL.createObjectURL(blob)
})
}

**Comment by DanielHauschildt** [comment]
Good catch. I will fix the engine requirements and publish a new version asap

**Comment by DanielHauschildt** [comment]
Should be fixed with 1.5.1 which was just published.
